CRM/Case/BAO/Query.php

```diff
@@ -260,6 +260,11 @@ public static function where(&$query) {
    * @param CRM_Contact_BAO_Query $query
    */
   public static function whereClauseSingle(&$values, &$query) {
+    if ($query->buildDateRangeQuery($values)) {
```

Comment:
On the others we don't hit this because the metadata handles it

Reply:
I see it used in these places 

```
CRM/Contact/BAO/Query.php:  public function buildDateRangeQuery($values) {
CRM/Pledge/BAO/Query.php:    if ($query->buildDateRangeQuery($values)) {
CRM/Member/BAO/Query.php:    if ($query->buildDateRangeQuery($values)) {
```
Maybe its just they were done early or something i dunno

Reply:
@seamuslee001 OK I'll merge this but maybe we can test after & see if required